packages/regl-worldview/src/commands/Cylinders.js

```diff
@@ -47,9 +49,8 @@ const { points, sideFaces, endCapFaces } = createCylinderGeometry(30, false);
 
 const cylinders = fromGeometry(points, sideFaces.concat(endCapFaces));
 
-const Cylinders = makeCommand<Cylinder>("Cylinders", cylinders, {
-  getHitmapProps,
-  getObjectFromHitmapId,
-});
-
-export default Cylinders;
+export default function Cylinders(props: { ...OptionalCommandProps, children: Array<Cylinder> }) {
+  return (
+    <Command getHitmap={createInstancedGetHitmap({ pointCountPerInstance: 1 })} {...props} reglCommand={cylinders} />
```

Comment:
🛑 Is this actually right? Each instance has `sideFaces.length + endCapFaces.length` elements (or 3 * that number of points). I don't see how that's being accounted for.

Screenshot-based tests for the hitmap logic would help with issues like this.

Reply:
I am not sure what is correct but I can check.

Reply:
For both cones and cylinders each point is just the centerpoint, so this is correct. Added some examples to make sure.